src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg'
+import styled from 'styled-components'
```

Comment:
My personal preference: 
```suggestion
import styled from 'styled-components/macro'
```